### PR TITLE
HP-60 - Remove modal-is-open-hip class from body element when sidebar modal is closed

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/HP-60

### CODE REVIEW

File: apps/hip/static/js/common.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function mobileModalOpen (target) {
```
Comment (vkurup) [inline apps/hip/static/js/common.js:1]: Minor comments, but you don't have to accept them. I think I'd prefer these to be named `openMobileModal` and `closeMobileModal` which makes it clear to me that by calling this function I am going to perform that action. A comment here saying something like `// target: element to open as a modal` or something might be helpful as well.

**Review by vkurup — APPROVED** [review]
Minor comments, feel free to ignore. I like how you split the code out to reuse it!

**Comment by Audiosutras** [comment]
> Minor comments, feel free to ignore. I like how you split the code out to reuse it!

Yea, I really like how we setup the javascript for this project. Sort of our own vanilla javascript version of react or vue.

File: apps/hip/static/js/common.js (hunk 1 above)
Comment (Audiosutras) [reply apps/hip/static/js/common.js]: done

**Pull request merged**